packages/angular-material/src/other/object.renderer.ts

```diff
@@ -65,7 +67,14 @@ export class ObjectControlRenderer extends JsonFormsControlWithDetail {
       props.schema,
       props.uischema.scope,
       props.path,
-      'Group',
+      () => {
+        const newSchema = cloneDeep(props.schema);
+        // delete unsupported operators
+        delete newSchema.oneOf;
+        delete newSchema.anyOf;
+        delete newSchema.allOf;
+        return generateDefaultUISchema(newSchema, 'Group');
```

Comment:
While this is the underlying implementation, we typically invoke it via `Generate.uischema`. Can you adapt accordingly?